torch/csrc/jit/python_ir.cpp

```diff
@@ -356,7 +356,7 @@ void initPythonIRBindings(PyObject * module_) {
 #undef CREATE_ACCESSOR
     // Tensor (t_) -- manually written to unwrap the variable into a tensor.
     .def("t_",[](Node & n, const char * name, torch::autograd::Variable v) {
-      return n.t_(Symbol::attr(name), std::move(v.data()));
+      return n.t_(Symbol::attr(name), v.data());
```

Comment:
We remove the `std::move(v.data())` uses in this file, because it leaves the `data_` field empty for `v` (and puts UndefinedTensorImpl in `v.data().impl_`), which causes segfault in the THPVariable destruction path because the AutogradMeta pointer is null in a UndefinedTensorImpl. Changing `std::move(v.data())` to `v.data()` fixes the issue.

Reply:
Update: Got confirmation from Peter - it's okay to remove `std::move` here.